src/style/card.css

```diff
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-items: center;
+
+  width: 200px;
+
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  margin-bottom: 40px;
+  max-height: 134px;
+
+  object-fit: contain;
+}
+
+.card__title {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: var(--main-dark-color);
+
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--ligth-gray-color);
+
+  margin-bottom: 16px;
+}
+
+.card__price-box {
+  display: flex;
+  justify-content: space-between;
+
+  margin-bottom: 16px;
+}
+
+.card__price-label {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--ligth-gray-color);
+}
+
+.card__price {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--main-dark-color);
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 166px;
+  height: 40px;
+
+  background: var(--detail-blue-color);
+  border-radius: 5px;
+}
+
+.card__button:hover {
+  background: #fff;
+  border: 1px solid var(--detail-blue-color);
```

Comment:
Look through the checklist again, please (9. [STYLES] - Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover)